<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="noir">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>250</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#noir{
background:noir;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_juge">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{font: 40pt  'Georgia';font-size:22px;

color :white;
}

QPushButton{
background: transparent;


}</string>
    </property>
    <property name="text">
     <string>Juges</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_audience">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{font: 40pt  'Georgia';font-size:22px;

color :white;
}

QPushButton{
background: transparent;


}</string>
    </property>
    <property name="text">
     <string>Audiences</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_affaires">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{font: 40pt  'Georgia';font-size:22px;
qproperty-icon:url(&quot;:archiver.png&quot;);
color :white;
}

QPushButton{
background: transparent;


}</string>
    </property>
    <property name="text">
     <string>Archives</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_salles">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{font: 40pt  'Georgia';font-size:22px;

color :white;
}

QPushButton{
background: transparent;


}</string>
    </property>
    <property name="text">
     <string>Salles </string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_avocat">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>560</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{font: 40pt  'Georgia';font-size:22px;

color :white;
}

QPushButton{
background: transparent;


}</string>
    </property>
    <property name="text">
     <string>Avocats</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="rouge">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#rouge{

background:#710100;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="back">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>10</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#back{

background:white;



}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_j">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>21</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_j{

background:#710100;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_aud">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>21</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_aud{

background:#710100;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_ar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>21</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_ar{

background:#710100;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_salle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>21</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_salle{

background:#710100;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_avocat">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>21</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_avocat{

background:#710100;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_home">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>21</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_home{

background:#710100;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_home">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{font: 40pt  'Georgia';font-size:22px;

color :white;
}

QPushButton{
background: transparent;


}</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_ihome">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_iarchive">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_ijuge">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_isalle">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_iavocat">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>540</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_iaudience">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>371</width>
      <height>601</height>
     </rect>
    </property>
    <property name="title">
     <string>ajouter modifier</string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>numero d'archive</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Type de crime</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>violence</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>vole</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>assasignation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>viole</string>
      </property>
     </item>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>194</width>
       <height>22</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>291</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>490</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Jugement Final</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>execution</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pecher Financier</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>prison</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>prison et Pecher Financier</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_ajouter_arch">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>560</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_modifier_arch">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>560</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>80</y>
      <width>391</width>
      <height>601</height>
     </rect>
    </property>
    <property name="title">
     <string>Afficher</string>
    </property>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>70</y>
       <width>341</width>
       <height>431</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>recherche</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_supprimer_arch">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>550</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>550</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Imprimer</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Par Defaut</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Numero archive</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>type crime</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>jugement Final</string>
      </property>
     </item>
    </widget>
   </widget>
   <zorder>back</zorder>
   <zorder>noir</zorder>
   <zorder>pushButton_juge</zorder>
   <zorder>pushButton_audience</zorder>
   <zorder>pushButton_affaires</zorder>
   <zorder>pushButton_salles</zorder>
   <zorder>pushButton_avocat</zorder>
   <zorder>rouge</zorder>
   <zorder>logo</zorder>
   <zorder>label_j</zorder>
   <zorder>label_aud</zorder>
   <zorder>label_ar</zorder>
   <zorder>label_salle</zorder>
   <zorder>label_avocat</zorder>
   <zorder>label_home</zorder>
   <zorder>pushButton_home</zorder>
   <zorder>label_ihome</zorder>
   <zorder>label_iarchive</zorder>
   <zorder>label_ijuge</zorder>
   <zorder>label_isalle</zorder>
   <zorder>label_iavocat</zorder>
   <zorder>label_iaudience</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
